### ISSUE TITLE
instanceof relation is strange

### ISSUE THREAD

**Issue by Anaphory** [clld/concepticon-data#468]
The `instanceof` relation is used in four places, eg. https://github.com/clld/concepticon-data/blob/master/concepticondata/conceptrelations.tsv#L276

However, the concept MID-AUTUMN FESTIVAL is just a very specific sub-concept of FESTIVAL, and the same holds for the other concepts related through `instanceof`, so I think you should either use `narrower` or define those terms.

My naïve understanding of `instanceof` in the absence of defined semantics is that it applies to eg. the relation between FOUR and NUMBER, or maybe to WORLD → PLACE (POSITION), but overall very rarely in the database, because generally a concept is a class, not an instance.

**Comment by tresoldi** [comment]
Indeed, this is one the points to bring up when explaining that there are similarities but Concepticon is not a lexical database like WordNet or a semantic network like BabelNet.

**Comment by LinguList** [comment]
So what would you propose as alternative, @Anaphory? Feel free to go ahead. You can also delete it. I added it when adding the Chinese festivals. I could've left them out, as you may think this is too special, but in fact the festivals occur in several Chinese questionnaires. We never went through all concepts in toto to search for which relations to add, but rather did it on an ad-hoc basis. I don't mind loosing those relations that seem strange, if most of you think they're not needed. I'd say: feel free to change.
